.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,37 @@
+on: [push, pull_request]
+name: Integration Tests
+
+concurrency:
+  group: ${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  unit:
+    strategy:
+      fail-fast: false
+      matrix:
+        suite: [ "docker_examples", "local_exec" ]
+        os: [ "ubuntu" ]
+        go: [ "1.16.x" ]
+    env:
+      COVERAGES: ""
+    runs-on: ${{ format('{0}-latest', matrix.os) }}
+    name: "[${{ matrix.suite }}] on ${{ matrix.os }} (go ${{ matrix.go }})"
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
```

Comment:
Do we have submodules here? It rings a bell but I cannot remember for what.

Reply:
We don't at the moment, I removed it 🙏